src/pages/Cart/CartPage.jsx

```diff
@@ -16,42 +17,22 @@ export const Cart = () => {
           <ErrorMessage message={"your cart is empty"} />
         </Link>
       ) : (
-      <div className="wishlist-card-container">
-        <ul className="cart-left-container">
-          <li>
-          {cartItem.map((item) => {
-            return <HorizontalCard key={item._id} item={item}/>
-        })}
-          </li>
-        </ul>
-        <div className="cart-right-container">
-          <div className="price-details-style">PRICE DETAILS</div>
-          <hr />
-          <div className="bill-details-row">
-            <div className="col-left-wd ">Price ({cartCount} items)</div>
-            <div className="col-right-wd  text-align-right">₹3000.00</div>
-          </div>
-          <div className="bill-details-row">
-            <div className="col-left-wd ">Discount </div>
-            <div className="col-right-wd  text-align-right primary-color">
-              ₹600.00
-            </div>
-          </div>
-          <div className="bill-details-row ">
-            <div className="col-left-wd ">Devilvery charges</div>
-            <span className="text-price-strike">₹99 </span>
-            <span className="primary-color">FREE</span>
-            <div className="col-right-wd  text-align-right"></div>
-          </div>
-          <div className="bill-details-row">
-            <div className="col-left-wd ">Total Amount</div>
-            <div className="col-right-wd text-align-right">₹2400.00</div>
+        <div className="wishlist-card-container">
+          <ul className="cart-left-container">
+            <li>
+              {cartItem && cartItem.map((item) => (
+                <HorizontalCard item={item} />
+              ))}
+            </li>
+          </ul>
+          <div className="cart-right-container">
+          {cartItem.length === 0 ? (
+              ""
+            ):(
+            <TotalPriceCard cartState={cartState}/>)}
```

Comment:
you can use && if you don't need the if part  like this
```(cartItem?.length === 0) && <TotalPriceCard cartState={cartState}/>```

Reply:
yeah, thanks for suggestion, will try to make changes in code.